packages/core/src/sdk/geolocation/useGeolocation.ts

```diff
@@ -1,27 +1,43 @@
 import { useEffect } from 'react'
 
 import { deliveryPromise } from 'discovery.config'
-import { useSession, validateSession, sessionStore } from 'src/sdk/session'
+import { validateSession, sessionStore } from 'src/sdk/session'
 
-export default function useGeolocation() {
-  const { isValidating: _, ...session } = useSession()
+const TIME_TO_VALIDATE_SESSION = 3000
```

Comment:
question: how was this time chosen?

Reply:
From my testing, but it's arbitrary since in the first load `useSession` validates 2 to 3 times the session.